.github/workflows/goreleaser.yml

```diff
@@ -38,7 +42,7 @@ jobs:
         run: echo "::set-output name=go::$(go version | cut -d ' ' -f 3)"
 
       - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v5
+        uses: goreleaser/goreleaser-action@336e29918d653399e599bfca99fadc1d7ffbc9f7 # v4.3.0
```

Comment:
Any reason not to use v5 of goreleaser? (I feel like we should probably just bite the bullet and pull all the updates / work out all the kinks in the release process now with the RC but that's not a strong opinion)

Reply:
I just copied `main` to keep it safe to run as main runs. I don't even like the file name much less the contents. We should revisit this (and include docker publish), but I want initial release to get done.